res/package/JABBERWOCKY

```diff
@@ -1 +1 @@
-'Twas brillig, and the slithy toves
Did gyre and gimble in the wabe;
All mimsy were the borogoves,
And the mome raths outgrabe.
\ No newline at end of file
+This station is conducting a test of theEmergency Broadcast System.

This is only a test.

The broadcasters of your area in
voluntary cooperation with the FCC and
other authorities have developed this
system to keep you informed in the
event of an emergency.

If this had been an actual emergency,
you would have been instructed where
to tune in your area for news and 
official information.

This concludes this test of the
Emergency Broadcast System.

```

Comment:
Spaces missing in "theEmergency", "involuntary", after periods, etc ?

Also, rename the file (and update `bin/manifest`) since it's no longer the Jabberwocky poem's introductory stanza?

Later on we should update the file to use phoneme-mode.

Reply:
I changed it for the echo output, but it's speaking incorrectly. I can fix it. I just wanted to show what was possible and also throw in an '80s reference!
This may seem silly, but I was unaware of the poem! I can rename it.